src/components/Buttons/myButton.stories.tsx

```diff
@@ -0,0 +1,394 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+import Button from "./myButton";
+import { ReactComponent as GoogleIcon } from "../../assets/vectors/jsx/HeaderSignInViewGoogle.svg";
```

Comment:
we should use absolute imports for components being imported from a different directory. Or for every import if possible